### PR TITLE
Doc adjustments and elimination of shadowed import warning

### PR DESCRIPTION
**Pull request by pdolega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # ReactSphere-reactive-beyond-hype
 Repo for presentation on ReactSphere: Reactive beyond hype
 
-### Requirements:
- * ammonite
+### Requirements (with install instructions:)tw
```
Comment (lbialy) [inline README.md]: typo on the end of line

Comment (pdolega) [reply README.md]: Fixed

File: codebase/commons/src/main/scala/com/virtuslab/Config.scala [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package com.virtuslab
 
-import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.config.{Config => TypesafeConfig, ConfigFactory}
```
Comment (pdolega) [inline codebase/commons/src/main/scala/com/virtuslab/Config.scala]: ```Warning:(3, 29) imported `Config' is permanently hidden by definition of object Config in package virtuslab
import com.typesafe.config.{Config, ConfigFactory}```

Comment (pdolega) [inline codebase/commons/src/main/scala/com/virtuslab/Config.scala]: ```Warning:(3, 29) imported `Config' is permanently hidden by definition of object Config in package virtuslab
import com.typesafe.config.{Config, ConfigFactory}```

**Pull request merged**